src/frontend/Komponenter/Behandling/Vurdering/ModalGjenbrukVilkårsvurdering.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Modal, BodyLong } from '@navikt/ds-react';
+import React, { FC } from 'react';
+
+const ModalGjenbrukVilkårsvurdering: FC<{
+    visModal: boolean;
+    settVisModal: React.Dispatch<React.SetStateAction<boolean>>;
+    gjenbrukVilkårsvurdering: () => void;
+}> = ({ visModal, settVisModal, gjenbrukVilkårsvurdering }) => {
+    if (!visModal) {
+        return null;
+    }
+
+    return (
+        <Modal
+            onClose={() => settVisModal(false)}
+            open={visModal}
+            aria-labelledby="modal-heading"
+            header={{ heading: 'Gjenbruk av vilkårsvurdering' }}
+        >
+            <Modal.Body>
+                <BodyLong>
+                    Er du sikker på at du vil gjenbruke vilkårsvurdering fra tidligere behandling?
+                    Inngangsvilkåret vil bli overskrevet.
+                </BodyLong>
+            </Modal.Body>
+            <Modal.Footer>
+                <Button
+                    type="button"
+                    onClick={() => {
+                        gjenbrukVilkårsvurdering();
+                        settVisModal(false);
+                    }}
+                >
+                    Gjenbruk
+                </Button>
+                <Button type="button" variant="secondary" onClick={() => settVisModal(false)}>
+                    Avbryt
+                </Button>
+            </Modal.Footer>
+        </Modal>
+    );
+};
+
+export default ModalGjenbrukVilkårsvurdering;
```

Comment:
Jeg er egentlig blitt fan av å exportere direkte der konstanten blir definert (named export) i stedet for å ta i bruk default exports. Grunnen er at når man skal sjekke hvor en komponent som er tatt i bruk så dukker defualt exporten opp i søket og skaper "støy". 

Såvidt jeg har forstått så finnes det egentlig ingen god grunn til å bruke default exports heller. Jeg har derfor lyst til at vi skal gå bort fra default exports til named exports generelt i kodebasen